### PR TITLE
Add required files for Package control

### PR DESCRIPTION
**Pull request by ayoub-benali**
Inspired by https://github.com/sublimelsp packages and https://packagecontrol.io/docs/submitting_a_package

### CODE REVIEW

**Comment by predragnikolic** [comment]
The reason why I think you need a `.no-sublime-package` is because of this bolded text.
> if including **executables** or shared libraries, add a .no-sublime-package file. Adding this file to the root of your repository will prevent Package Control from keeping your package packed as a .sublime-package file in ST3.

I noticed that you have this executable 
https://github.com/scalameta/metals-sublime/blob/master/coursier

I am not sure if this line will throw an error, when wrapped as `.sublime-package` :) 
https://github.com/scalameta/metals-sublime/blob/master/plugin.py#L9

**Comment by ayoub-benali** [comment]
@predragnikolic thanks for the info, I haven't noticed that part of the doc 😇
Do you think the difference between the repo name and package name in the json will be ok for package control ?

**Comment by predragnikolic** [comment]
https://packagecontrol.io/docs/submitting_a_package

> 2. Pick a Name
    Try not to use the word Sublime in your package name. 

So to answer your question. `LSP-metals` is fine :)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (predragnikolic) [inline package.json]: package control don't require a `package.json` file :)

**Pull request merged**